.github/ISSUE_TEMPLATE/story.md

```diff
@@ -38,13 +38,10 @@ As a _, so that _, I need _.
 - [ ] Code quality tasks completed
   - [ ] Code refactored for clarity and no design/technical debt
   - [ ] Adhere to separation of concerns; code is not tightly coupled, especially to 3rd party dependencies
-  - [ ] Code is reviewed or developed by pair; 1 approval is needed but consider requiring an outside-the-pair reviewer
-  - [ ] Code quality checks passed
```

Comment:
I'm curious to know why this needs to get deleted.

Reply:
Removing this doesn't mean we're changing our process.  These were removed because they are automatically accounted for when doing the PR process, and therefore accounted for in the existing `Source code is merged to the main branch` DoD item.

Reply:
Just wondering - I agree @halprin that the PR review process implies being reviewed by a pair, but I'm wondering if the outside reviewer part is implied. I know the PR creator can't approve by function, but git doesn't stop the other collaborator(s) from approving (unless there's something about co-authors Ii don't know).

Reply:
@tjohnson7021, you've got it right.  A collaborator could unilaterally approve it.  And yes, I think it should continue to be ideal that reviewers be outside the people who worked on the PR.  You aren't always guaranteed to be able to get this, but that's OK.